attributes/default.rb

```diff
@@ -14,6 +14,7 @@
 default["td_agent"]["in_http"]["enable_api"] = true
 default["td_agent"]["version"] = "2.2.0"
 default["td_agent"]["pinning_version"] = false
+default["td_agent"]["manage_repo"] = true
```

Comment:
duplicates functionality of `default['td_agent']['skip_repository'] = false`

Reply:
oops, thanks :D